samples/dnn/facial_landmarks_ie.cpp

```diff
@@ -0,0 +1,195 @@
+     "{ landmweights         | |     Full path to a facial Landmarks detection model weights file (for example, .bin file).}"
+     "{ input                | |     Full path to an input image or a video file. Skip this argument to capture frames from a camera.}"
+     "{ b                    | |     The color to draw face rectangle and landmarks in format \"(b, g, r)\".}"
+     "{ g                    | |     The color to draw face rectangle and landmarks in format \"(b, g, r)\".}"
+     "{ r                    | |     The color to draw face rectangle and landmarks in format \"(b, g, r)\".}"
+     "{ faceframework        | |     Optional name of an origin framework of the facedetector. Detect it automatically if it does not set. }"
+     "{ landmframework       | |     Optional name of an origin framework of the landmarkdetector. Detect it automatically if it does not set. }"
+     "{ facebackend          | 0 |   Choose one of computation backends for facedetector: "
+                                         "0: automatically (by default), "
+                                         "1: Halide language (http://halide-lang.org/), "
+                                         "2: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+                                         "3: OpenCV implementation }"
+     "{ landmbackend         | 0 |   Choose one of computation backends for facedetector: "
+                                         "0: automatically (by default), "
+                                         "1: Halide language (http://halide-lang.org/), "
+                                         "2: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+                                         "3: OpenCV implementation }"
+     "{ facetarget           | 0 |   Choose one of target computation devices for facedetector: "
+                                         "0: CPU target (by default), "
+                                         "1: OpenCL, "
+                                         "2: OpenCL fp16 (half-float precision), "
+                                         "3: VPU }"
+     "{ landmtarget          | 0 |   Choose one of target computation devices for landmarkdetector: "
+                                         "0: CPU target (by default), "
+                                         "1: OpenCL, "
+                                         "2: OpenCL fp16 (half-float precision), "
+                                         "3: VPU }"
+        );
+
+    parser.about("Use this script to run classification deep learning networks using OpenCV.");
+    if (argc == 1 || parser.has("help"))
+    {
+        parser.printMessage();
+        return 0;
+    }
+
+    //Parsing input arguments
+    const std::string facexmlPath = parser.get<std::string>("facestruct");
```

Comment:
If you use Camel case for variable naming please follow standard rules for this:
CamelCase Notation
CamelCase (sometimes called "CamelBack") notation is the practice of writing compound words or phrases in which the parts are joined without spaces, with each part's initial letter capitalized and the first letter in either upper or lower case. For example: SampleDataTable, computeExpr